e2e/__tests__/__snapshots__/chaiAssertionLibrary.ts.snap

```diff
@@ -0,0 +1,91 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`chai assertion errors should display properly 1`] = `
+FAIL __tests__/chai_assertion.js
+  ● chai.js assertion library test › expect
+
+    assert.(received, expected)
```

Comment:
This doesn't look right, I think there should be something like `assert.equal` here?

Reply:
ah, terribly sorry, will modify now.

Reply:
@jeysal ,
I checked `assertionErrorMessage` method in both `jasmine` and `circus`.
in below line is trying to extract the operator.
`const {expected, actual, generatedMessage, message, operator, stack} = error;`
But `error` does not have an `operator` attribute.
The stringified version of the `AssertionError` is as follows.
```json
{
    "actual": "hello world",
    "expected": "hello sunshine",
    "message": "expected 'hello world' to equal 'hello sunshine'",
    "name": "AssertionError",
    "showDiff": true,
    "stack": "AssertionError: expected 'hello world' to equal 'hello sunshine'
    at Object.equal (/ruwan/repos/my_testing_ground/jest_testing/chai.spec.js:5:35)
    at Object.asyncJestTest (/ruwan/repos/jest_unit/jest/packages/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/ruwan/repos/jest_unit/jest/packages/jest-jasmine2/build/queueRunner.js:43:12)
    at new Promise (<anonymous>)
    at mapper (/ruwan/repos/jest_unit/jest/packages/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/ruwan/repos/jest_unit/jest/packages/jest-jasmine2/build/queueRunner.js:73:41)
    at <anonymous>"
}
```
Only place there is a hint is in the first line of the stack trace. 
Do you want me to give a try to extract operator from stack trace?

Reply:
@jeysal ,
It seems the `chai` library doesn't support `operator` yet.
(I would like to try to create a PR for this in `chai`. 😄 ).
https://github.com/chaijs/chai/issues/1252

Reply:
I guess we should guard against `operator` being undefined anyway even if chai starts supporting it at some point. Maybe we should just hide the whole `assert` line in that case? If the operator is unknown, it doesn't provide any value anyway (There was an assertion with an expected value and an actual value, who would have thought 😄). Usually you'll still see the operator in the stack trace at least.

Reply:
@jeysal,
I can do that modification. I'll hide the particular message if the operator is not defined.